src/runtime_src/core/common/ishim.h

```diff
@@ -72,6 +72,9 @@ struct ishim
   virtual void
   get_bo_properties(xclBufferHandle boh, struct xclBOProperties *properties) const = 0;
 
+  virtual void
+  get_device_info(struct xclDeviceInfo2 *info) const = 0;
+
```

Comment:
I don't think this is needed for the changes you've made

Reply:
true this is not needed for that changes i made but since we have exposed all other shim api's we exposed this also. this can be used at other places also in future. 
@stsoe,  any specific reason of not adding this in ishim?

Reply:
Yes, I would like to see when this is needed and why.  I think device info should come from individual query requests.  For example windows does not truly populate this struct but does provide query requests.